README.md

```diff
@@ -17,158 +17,30 @@ Add this line to your application's Gemfile:
-To specify the US region do:
-`config.region = :us`
-
-To specify the CA region do:
-`config.region = :ca`
-
-See https://documentation.onfido.com/#regions for supported regions.
-
-## Usage
-
-You can make API calls by using an instance of the `API` class:
-
-```ruby
-onfido = Onfido::API.new
-```
-
-You can also set an API key as follows, instead of in the initializer configuration:
-
-```ruby
-onfido = Onfido::API.new(api_key: '<YOUR_API_KEY>')
+onfido = Onfido::API.new(
+  api_key: ENV['ONFIDO_API_KEY'],
+  # Supports :eu, :us and :ca. Previously defaulted to :eu.
+  region: :eu
+)
 ```
 
-### Resources
-
 All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.all` to fetch all resources.
 
-**Note:** *All param keys should be symbols e.g. `{ report_names: ['document'] }`*
-
-#### Applicants
-
-Applicants are the object upon which Onfido checks are performed.
-
-```ruby
-onfido.applicant.create(params)                  # => Creates an applicant
-onfido.applicant.update('<APPLICANT_ID>', params)  # => Updates an applicant
-onfido.applicant.destroy('<APPLICANT_ID>')         # => Schedule an applicant for deletion
-onfido.applicant.restore('<APPLICANT_ID>')         # => Restore an applicant scheduled for deletion
-onfido.applicant.find('<APPLICANT_ID>')            # => Finds a single applicant
-onfido.applicant.all                             # => Returns all applicants
-```
-
-**Note:** Calling `onfido.applicant.destroy` adds the applicant and all associated documents, photos, videos, checks, and reports to the deletion queue. They will be deleted 20 days after the request is made. An applicant that is scheduled for deletion can be restored but applicants that have been permanently deleted cannot.
-See https://documentation.onfido.com/#delete-applicant for more information.
-
-#### Documents
-
-Some report types require identity documents (passport, driving licence etc.) in order to be processed.
-
-```ruby
-onfido.document.create(applicant_id: '<APPLICANT_ID>', file: <FILE>, type: 'passport') # => Creates a document
-onfido.document.find('<DOCUMENT_ID>')      # => Finds a document
-onfido.document.download('<DOCUMENT_ID>')  # => Downloads a document as a binary data
-onfido.document.all('<APPLICANT_ID>')      # => Returns all documents belonging to an applicant
-```
-
-**Note:** The file parameter must be a `File`-like object which responds to `#read` and `#path`.
-Previous versions of this gem supported providing a URL to a file accessible over HTTP or a path
-to a file in the local filesystem. You should instead load the file yourself and then pass it in
-to `#create`.
-
-See https://documentation.onfido.com/#document-types for example document types.
-
-#### Live Photos
-
-Live photos are images of the applicant’s face, typically taken at the same time as documents are provided. These photos are used to perform Facial Similarity Photo reports on the applicant.
-
 ```ruby
```

Comment:
Add a line before this code example saying something like: "For example, to create an applicant:"